docs/nor_gate_method.md

```diff
@@ -0,0 +1,66 @@
+---
+layout: default
+title: NOR gate method
+comments: true
+nav_order: 6
+---
+
+# NOR gate method to simplify boolean expressions
+{: .no_toc}
+
+## Table of contents
+{: .no_toc .text-delta }
+
+1. TOC
+{:toc}
+
+---
+
+## Introduction
+[NOR Gates](https://learn.circuitverse.org/docs/universal_gates.html#NOR-gate) are universal gates. By the virtue of functional completeness, NOR Gates can be used to fully represent a given boolean expression. This simplifies the expression such that only one standard gate is used throughout.  
+
+## Functional completeness
+Functional completeness is a property pertaining to boolean logic, which states that a functionally complete boolean operator can express all possible truth tables by representing itself in a boolean expression. That is, any given boolean expression can be completely represented by using the a functionally complete boolean operator. 
+
+For example, NOR gates can be used to implement the [NOT gate](https://learn.circuitverse.org/docs/universal_gates.html#implementing-not-gate-1), the [OR gate](https://learn.circuitverse.org/docs/universal_gates.html#implementing-or-gate-1) and the [AND gate](https://learn.circuitverse.org/docs/universal_gates.html#implementing-and-gate-1). 
```

Comment:
-1 should be removed from all the links as they are for Nand Gates. We are implementing these gates through NOT not NAND. Please update the links.

Reply:
Whoops! sorry :sweat_smile: will update it.